examples/regression/e2e/features/issue_4285.feature

```diff
@@ -0,0 +1,9 @@
+@check_issue_4005
```

Comment:
Small comment: I think you meant 4285 instead of 4005 on this line and the next.